.babelrc

```diff
@@ -1,15 +1,70 @@
 {
-  "presets": [
-    ["latest", {
-      "es2015": { "loose": true }
-    }],
-    "stage-2",
-    "react"
-  ],
-  "plugins": [
-    "dev-expression",
-    "add-module-exports",
-    "transform-object-assign",
-    ["transform-react-remove-prop-types", { "mode": "wrap" }]
-  ]
+  "env" : {
+    "development": {
+      "presets": [
```

Comment:
if we don't want to use a babel 7, `.babelrc.js`, then we should just make this a preset like https://github.com/relay-tools/react-router-relay/blob/v0.14.0/tools/latestPreset.js to avoid duplicating the code a bunch